index.html

```diff
@@ -1111,6 +1106,38 @@ <h2 id="state-transitions" class="informative">State transitions</h2>
 </section>
 
 <section>
+  <h2>PaymentRequest and iframes</h2>
+  <p>
+    There are some circumstances where a cross-origin <a>iframe</a> wants to make a payment
+    request. A cross-origin iframe needs explicit permission from the embedding page to invoke
+    the payment request API.
+  </p>
+  <p>
+    The <a>HTMLIFrameElement</a> is extended with an <dfn><code>allowpaymentrequest</code></dfn>
+    content attribue. <a><code>allowpaymentrequest</code></a> is a <a>boolean attribute</a>.
```

Comment:
This isn't really what we've discussed. Unless there was subsequent conversation outside of the issue, the proposal (which didn't have any disagreement) was to extend the already-defined "permissions" attribute with a new "payment" value.